pkg/utils/kubelessutil.go

```diff
@@ -684,6 +684,25 @@ func EnsureFuncDeployment(client kubernetes.Interface, funcObj *kubelessApi.Func
 		}
 	}
 
+	// Add soft pod anti affinity
+	dpm.Spec.Template.Spec.Affinity = &v1.Affinity{
```

Comment:
can you pre-check if `dpm.Spec.Template.Spec.Affinity` is nil before setting the value? That way people would be able to overwrite it if needed.

Reply:
Good idea, added.